blazor/pdfviewer/how-to/load-office-files.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: Load office files in Blazor PDF Viewer | Syncfusion
+description: Learn here all about how to load the microsoft office files like powerpoint in the Syncfusion Blazor PDF Viewer component and more.
+platform: Blazor
+control: PDF Viewer
+documentation: ug
+---
+
+# Load Microsoft Office files in Blazor PDF Viewer Component
+
+The PDF Viewer library allows you to load Microsoft office files such as powerpoint, excel, word and image using the success event of uploader.
```

Comment:
The PDF Viewer library allows you to load Microsoft office files such as PowerPoint, Excel, Word, and image using the success event of the uploader.